docs/guides/using-the-network-shim.md

```diff
@@ -0,0 +1,193 @@
+}
+```
+
+#### `./cypress.json`
+
+Interactive runs are not formally supported by the Network Shim, because `cypress open` (interactive mode) and `cypress run` have different behaviour regarding the plugin events. However, cypress is attempting to smooth things over and by setting `experimentalInteractiveRunEvents` to true it should theoretically be possible to at at least use stub mode in interactive mode, for example:
+
+```json
+{
+    "baseUrl": "APP URL",
+    "projectId": "PROJECT ID",
+    "testFiles": "TEST FILES",
+    "experimentalInteractiveRunEvents": true,
+    "video": false
+}
+```
+
+#### `./cypress/plugins/index.js`
+
+For most DHIS2 cypress setups, using the `enableAutoLogin` setup helps managing authentication conveniently. In turn, the `enableAutoLogin` setup utility requires the `chromeAllowXSiteCookies` when using Chrome. In most cases it makes sense to always use this plugin alongside the `networkShim`.
+
+The `networkShim` plugin itself doesn't require the `config` object, but it does accept a second `options` argument. For most DHIS2 platform apps the default options will be correct so there will be no need to pass a second argument.
+
+```js
+const {
+    networkShim,
+    chromeAllowXSiteCookies,
+} = require('@dhis2/cypress-plugins')
+
+/*
+ * const networkShimOptions = {
+ *     hosts: [
+ *         'https://main-api.com',
+ *         'https://secondary-api.com',
+ *     ],
+ *     staticResources: [
+ *         '/endpoint-which-always-return-the-same-data',
+ *         '/another-endpoint-which-always-return-the-same-data',
+ *     ]
+ * }
+ */
+
+module.exports = (on, config) => {
+    networkShim(on /*, networkShimOptions */)
+    chromeAllowXSiteCookies(on, config)
+}
+```
+
+#### `./cypress/support/index.js`
+
+The `enableNetworkShim` setup accepts no arguments and in order to execute a cypress run in a DHIS2 platform app. It's recommended that most apps use the `enableAutoLogin` setup utility to manage automatic authentication, which means the file should look like this:
+
+```js
+import { enableAutoLogin, enableNetworkShim } from '@dhis2/cypress-commands'
+
+enableAutoLogin()
+enableNetworkShim()
+```
+
+### Package scripts and dependencies
+
+#### `./package.json`
+
+The following `devDependencies` are required:
+
+```json
+"devDependencies": {
+    "@dhis2/cli-utils-cypress": "^#.#.#",
+    "@dhis2/cypress-commands": "^#.#.#",
+    "@dhis2/cypress-plugins": "^#.#.#",
+},
+```
+
+And it is recommended to add some convenience commands to the `scripts` section, for example:
+
+```json
+"scripts": {
+    "cy:start": "BROWSER=none yarn start",
+    "cy:open": "wait-on 'http-get://localhost:3000' && cypress open",
+    "cy:run": "wait-on 'http-get://localhost:3000' && cypress run",
+    "cy:capture": "concurrently 'yarn cy:start' 'yarn cy:run --env networkMode=capture' --kill-others --success first",
+    "cy:stub": "concurrently 'yarn cy:start' 'yarn cy:run --env networkMode=stub' --kill-others --success first",
```

Comment:
Personally I'd go with the start-server-and-test module. Cypress recommends it in their docs, it's maintained by a cypress maintainer, and it seems easier to grasp to a newcomer to me.

Concurrently requires more in-depth knowledge to use correctly.

Reply:
I'll add it as a recommendation below